sample.env

```diff
@@ -77,3 +77,11 @@ NODE_ENV=development
 PORT=3000
 FCC_ENABLE_SWAGGER_UI=true
 FCC_ENABLE_DEV_LOGIN_MODE=true
+
+# Email
+EMAIL_PROVIDER=nodemailer # use ses in production
+# SES_ID needs to be empty for now, since the api-server will use SES in testing
+# if it is set.
```

Comment:
In the new api, yeah, but the old api uses SES if `SES_ID` is truthy.

https://github.com/freeCodeCamp/freeCodeCamp/actions/runs/5724886824/job/15513669234

failed because it tried to use SES with an invalid key.